src/main/java/com/ehealth/mpi/config/AppConfiguration.java

```diff
@@ -0,0 +1,80 @@
+package com.ehealth.mpi.config;
+
+import java.beans.PropertyVetoException;
+import java.util.Properties;
+
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import com.ehealth.mpi.dao.IPatientDAO;;
+
+@Configuration
+@ComponentScan(basePackages = {
+		"com.ehealth.mpi" }, excludeFilters = @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, value = SpringWebConfig.class))
+@EnableTransactionManagement
+@EnableJpaRepositories(basePackageClasses = IPatientDAO.class)
+@PropertySource("classpath:config.properties")
+public class AppConfiguration {
+
+	@Value("${driver}")
+	private String driver;
+	@Value("${url}")
+	private String url;
+	@Value("${user}")

```

Comment:
This line causes a key conflict between config.properties var and namesake env var. The keys in config.properties should be unique, ie user.db, password.db, etc.